### PR TITLE
Publish demo 9 summary

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Publish demo 9 summary. Also add a publication hour to 1.7.8.0 release note.

### CODE REVIEW

File: news/_posts/public-oss-demo/2021-10-07-public-oss-demo-9-2021.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- [00:00](https://www.youtube.com/watch?v=x37-QOockEo) Beginning of the session, introduction
+- [02:35](https://www.youtube.com/watch?v=x37-QOockEo&t=155) Next PrestaShop version by @eternoendless
+- [32:38](https://www.youtube.com/watch?v=x37-QOockEo&t=1958) Product page discovery by @MatShir and @TristanLDD
+- [47:20](https://www.youtube.com/watch?v=x37-QOockEo&t=2840) Content style guide by @Julievrz and @prestascott
+- [01:01:29](https://www.youtube.com/watch?v=x37-QOockEo&t=3689) Questions & Answers
+
+
+You can use the timestamps to jump directly to the parts you want to watch.
+
+[Watch the replay on YouTube](https://www.youtube.com/watch?v=x37-QOockEo){: .btn .btn-lg .btn-info}
```
Comment (Julievrz) [inline news/_posts/public-oss-demo/2021-10-07-public-oss-demo-9-2021.md]:
```suggestion
[Watch replay on YouTube](https://www.youtube.com/watch?v=x37-QOockEo){: .btn .btn-lg .btn-info}
```

File: news/_posts/public-oss-demo/2021-10-07-public-oss-demo-9-2021.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+{: .text-center}
+
+Also you can find [here](https://docs.google.com/presentation/d/1PHPfuagoGRCDlXor4rf7h8W54vb0GvcobDJTWYmHf2Y/edit?usp=sharing) the slides used during the session.
```
Comment (Julievrz) [inline news/_posts/public-oss-demo/2021-10-07-public-oss-demo-9-2021.md]:
```suggestion
Also, you can take a look at the slides presented during the demo [here](https://docs.google.com/presentation/d/1PHPfuagoGRCDlXor4rf7h8W54vb0GvcobDJTWYmHf2Y/edit?usp=sharing).
```

File: news/_posts/public-oss-demo/2021-10-07-public-oss-demo-9-2021.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+As mentioned by @eternoendless, our plan for the next versions of PrestaShop is a proposal. We are looking for feedback and ideas about it, so please join us on [Slack](https://www.prestashop-project.org/slack/) to let us know what you think about what was presented!
```
Comment (Julievrz) [inline news/_posts/public-oss-demo/2021-10-07-public-oss-demo-9-2021.md]:
```suggestion
As mentioned by @eternoendless, our plan for the next versions of PrestaShop is a proposal. We are looking for feedback and ideas about it, so please join us on [Slack](https://www.prestashop-project.org/slack/) to let us know what you think! 
```

File: news/_posts/public-oss-demo/2021-10-07-public-oss-demo-9-2021.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+## Next session in November
+
+The next session will be streamed live on [**Wednesday, November 3, 2021 at 4pm CEST**](https://www.youtube.com/watch?v=jwOy3gx-MNs). See you then!
```
Comment (Julievrz) [inline news/_posts/public-oss-demo/2021-10-07-public-oss-demo-9-2021.md]:
```suggestion
Save the date: the next demo will be streamed live on [**Wednesday, November 3, 2021, at 4 pm CEST**](https://www.youtube.com/watch?v=jwOy3gx-MNs). See you then!
```

**Review by Julievrz — COMMENTED** [review]
LGTM, only a few suggestions. :-)

**Comment by matks** [comment]
@Julievrz thank you for review

**Pull request merged**